requirements.txt

```diff
@@ -1,4 +1,4 @@
-raiden==3.0.0rc9
+git+https://github.com/raiden-network/raiden.git@a5bdb80e0750901646cc3b7691bb1c9647ff5cdf#egg=raiden
```

Comment:
I'm not sure if we can create an Arbitrum release for Raiden, given that the app is broken. We would have to implement all the modifications for Arbitrum to create a proper release.

Reply:
This is !@#$

Reply:
I can think of 2 alternatives, and none of them sound great:

1. Implement the changes for Arbitrum in the python client, even though it will not be running on Arbitrum, and then we can create a release.

2. We implement on the services the part of the client we are importing, ending the dependency, and cutting the strings to the python client.

Both options demand a considerable amount of extra effort.

Reply:
---

We agreed to use a git tag on the `raiden` repo to ensure the commit will not be lost due to the GitHub garbage collector.